lib/Business/PayPal/API/GetTransactionDetails.pm

```diff
@@ -301,3 +349,13 @@ at your option, any later version of Perl 5 you may have available.
 
 
 =cut
+
+# Local Variables:
+#   Mode: CPerl
+#   indent-tabs-mode: nil
+#   cperl-indent-level: 4
+#   cperl-brace-offset: 0
+#   cperl-continued-brace-offset: 0
+#   cperl-label-offset: -4
+#   cperl-continued-statement-offset: 4
+# End:
```

Comment:
From what I gather these are hints to emacs?  I don't think they need to be in this file.  I'd rather see a .perltidyrc file included with the distribution so that all of the formatting requirements can be in one place.  I can add one once this pull request has been merged.